src/v/datalake/proto_to_arrow_struct.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include "datalake/errors.h"
```

Comment:
this commit won't build because this header file isn't introduced until the last commit